client/galaxy/scripts/mvc/tag.js

```diff
@@ -133,7 +135,70 @@ var TagsEditor = Backbone.View
 });
 
 // =============================================================================
+
+var WorkflowTagsEditor = TagsEditor.extend({
```

Comment:
Yeah or you can add flags to govern the closing behavior or any other preferences, we have something similar here: https://github.com/galaxyproject/galaxy/blob/dev/client/galaxy/scripts/mvc/ui/ui-modal.js#L15.
This would avoid inheritance and would let us use combinations of options.